.github/workflows/lint-test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x]
```

Comment:
can we add node 18

Reply:
done here: https://github.com/MetaMask/eth-simple-keyring/pull/109/commits/b68205c2a68cd3e3de80cbd6b192f6fc8465f4f6